sdk/cognitivelanguage/ai-language-text/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+_ Add support logging API warnings
```

Comment:
```suggestion
- Add support logging API warnings
```

Also might be good to clarify that they are warnings from the response headers. Could be confused with document-level warnings which we are not logging.

Reply:
Good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/24964/commits/c85bceb504abdfd0d10b4fbe0e96d27dc9f33ede.